step2/try3.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+func main() {
+	//現在時刻をナノ秒で表し変数に代入
+	t := time.Now().UnixNano()
+	//現在時刻を乱数の種として設定
+	rand.Seed(t)
+	s := rand.Intn(6) /*0-5の乱数*/
+
+	switch s + 1 {
+	case 1:
+		fmt.Println("凶")
+	case 2, 3:
+		fmt.Println("小吉")
+	case 4,5:
+		fmt.Println("中吉")
+	default:
+		fmt.Println("大吉")
+	}
```

Comment:
そんなにクリティカルじゃないけど、この方が可読性高そうだね！

```suggestion
	switch s {
	case 0:
		fmt.Println("凶")
	case 1, 2:
		fmt.Println("小吉")
	case 3, 4:
		fmt.Println("中吉")
	default:
		fmt.Println("大吉")
	}
```

Reply:
課題内容がサイコロの値でおみくじを作ろうという内容だったので、1〜６の数字を使用したのですが、実行したら結果が表示されるだけなので、0〜５のまま記述した方がわかりやすいですね、、、、
修正しておきました！